Mathlib/FieldTheory/IsAlgClosed/AlgebraicClosure.lean

```diff
@@ -198,3 +198,25 @@ instance {L : Type*} [Field k] [Field L] [Algebra k L] [Algebra.IsAlgebraic k L]
   isAlgClosed := inferInstance
 
 end AlgebraicClosure
+
+namespace IntermediateField
+
+variable {K L : Type*} [Field K] [Field L] [Algebra K L] (E : IntermediateField K L)
+
+theorem AdjoinSimple.normal_algebraicClosure {x : L} (hx : IsAlgebraic K x) :
```

Comment:
I've always found the name AdjoinSimple (or adjoin_simple) weird and am tempted to rename it AdjoinSingleton.

And I think this and the next lemma should be proven by combining [IsAlgClosure.normal](https://leanprover-community.github.io/mathlib4_docs/Mathlib/FieldTheory/IsAlgClosed/Basic.html#IsAlgClosure.normal) and `Algebra.IsAlgebraic.trans`.